docs/working-with-user.md

```diff
@@ -74,51 +74,58 @@ To retrieve a single user you need his UUID (for example:
 94bbe688-4b1e-4e4e-80e7-e5ba5c4d6db4):
 
 ```java
-OsiamConnector oConnector = [Retrieving an OsiamConnector](create-osiam-connector.md)
-AccessToken accessToken = [Retrieving an AccessToken](login-and-getting-an-access-token.md#retrieving-an-accesstoken)
-User user = oConnector.getUser(<USER_UUID>, accessToken);
+User user = osiamConnector.getUser(<USER_UUID>, accessToken);
 ```
 
+It is possible to filter the attributes of the returned resource by specifying the attributes to return as parameters 
+to the method:
+
+ ```java
+User user = osiamConnector.getUser(<USER_ID>, accessToken, "userName", "displayName","meta.created");
+ ```
+
 (please consider the possible runtimeException which are explained in the
 Javadoc)
 
-# Retrieve the current logged in user by his access token
+# Retrieve the currently logged in user by his access token
 
-If you are logged in with the client scope and you try to retrieve the current
-logged user you will retrieve a ConflictException.
+To retrieve the user that the currently held access token belongs to, the `getMe()` method is avaiable:
 
-```sh
-OsiamConnector oConnector = [Retrieving an OsiamConnector](create-osiam-connector.md)
-AccessToken accessToken = [Retrieving an AccessToken](login-and-getting-an-access-token.md#retrieving-an-accesstoken)
+```java
 // retrieves the complete currently logged in User.
-User user = oConnector.getMe(accessToken);
+User user = osiamConnector.getMe(accessToken);
+```
+It is possible to filter the attributes of the returned user by supplying the wanted attributes as parameters
+to the method:
+
+```java
+User user = osiamConnector.getUser(<USER_ID>, accessToken, "userName", "displayName","meta.created");
 ```
 
 # Retrieve all Users
 
 If you want to retrieve all users you can call the following method:
 
 ```java
-OsiamConnector oConnector = [Retrieving an OsiamConnector](create-osiam-connector.md)
-AccessToken accessToken = [Retrieving an AccessToken](login-and-getting-an-access-token.md#retrieving-an-accesstoken)
-SCIMSearchResult<User> searchResult = oConnector.getAllUsers(accessToken);
+SCIMSearchResult<User> users = oConnector.getAllUsers(accessToken);
 int numberOfUsers = searchResult.getTotalResults();
 for (User actUser : searchResult.Resources()) {
 	//...
 }
-//...
+```
+It is possible to filter the attributes of the returned users by supplying the attributes as parameters to the merhod:
+
+```java
+SCIMSearchResult<User> users = oConnector.getAllUsers(accessToken, "userName", "meta.created");
```

Comment:
`org.osiam.client.OsiamConnector#getAllUsers` returns `List<User>` not `SCIMSearchResult<User>`